mixins/demo/visibility-test-list.js

```diff
@@ -0,0 +1,73 @@
+import '../../components/button/button.js';
+import './visibility-test-item.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { VisibilityMixin } from '../visibility-mixin.js';
+
+export class VisibilityTestList extends VisibilityMixin(LitElement) {
+
+	static get properties() {
+		return {
+			_changeTracker: { type: Number, reflect: false },
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				display: block;
```

Comment:
I'm noticing a snap when the list shows/hides. Changing this to be `grid` seems to prevent this from happening.